index.js

```diff
@@ -237,6 +237,28 @@ const execa = (command, args, options) => {
 		return Promise.reject(error);
 	}
 
+	const originalKill = spawned.kill.bind(spawned);
+	spawned.kill = (signal, options) => {
+		if (signal === undefined || signal === 'SIGTERM') {
+			const retryAfter = options !== undefined &&
+				['number', 'string'].includes(typeof options.retryAfter) ?
+				Number(options.retryAfter) :
+				2000;
+
+			setTimeout(() => {
```

Comment:
`setTimeout()` should only be called after `originalKill()` has returned `true`. This is in case `signal` is erroneous (which will throw an error) or there is some permissions error (which will emit an `error` event). In both cases, we don't want to repeat `originalKill()`.

Reply:
Good idea, thanks: b39acfdb29d37f064b448e54d50d5dad3610f6e8